aws-datastore/src/main/java/com/amplifyframework/datastore/model/ModelProviderLocator.java

```diff
@@ -33,16 +34,21 @@
  * There is no compile-time guarantee that this class actually exists, or that it has that method,
  * or that we can use that method. But in the "happy path," this simplifies the amount of complexity
  * needed to begin using the DataStore. This utility class is used from
- * {@link AWSDataStorePlugin#AWSDataStorePlugin()}.
+ * {@link com.amplifyframework.datastore.AWSDataStorePlugin#AWSDataStorePlugin()}.
```

Comment:
I think you can just import it to avoid typing out the full name

Reply:
I had it like that originally, however, then `import com.amplifyframework.datastore.AWSDataStorePlugin` appears at the top, and makes it look like a source-code dependency. The relationship is actually the _opposite_ in reality, the plugin uses the locator, not vice-versa. To avoid ambiguity or misconception, I left it this way.